### DISCUSSION TITLE
Relative path in arguments

### DISCUSSION THREAD

**Discussion by WanderFox** [KindaBrazy/LynxHub#11] [Ideas]
Hi! It is will be great to make possibility to add relative path. For example: I'm changing output,input and temp directoryes for ComfyUI and I need to change lynx-user.bat manually

**Comment by KindaBrazy** [comment]
Hi! Could you explain a bit more?
Do you mean using a relative path for the ComfyUI folder, like `./output`? That way, you could move the ComfyUI folder more easily?

**Reply by WanderFox** [reply]
Yeah. Now I'm using path like "../../temp" "../../output/ComfyUI". It's important when you keep your AI-staff like a ComfyUI and other on external SSD. I think, it will be enough to type the path in Arguments by yourself, without target folder selection. It can give more flexibility. "Do you want to use a full path? Easy! Relative one? Yep - here you are!"

**Reply by KindaBrazy** [reply]
I’ll work on adding a way to choose in explorer and calculate relative paths, allowing you to switch between relative and absolute paths. If that gets tricky, I'll add a button to toggle between selecting a folder and manually typing the path.

For now, you can manually edit the `RendererMethods_xxxxxx.mjs` file in: `C:\Users\USERNAME\Documents\LynxHub\Modules\LynxHub-Module-Offline-Container\scripts`

Search for `--output-directory`, change `type: 'Directory'` to `type: 'Input'`, and restart LynxHub.

**Comment by KindaBrazy** [comment]
I've implemented switching between relative and absolute paths, along with real-time autocomplete for relative paths as you type.
c1d03755b7dcf31b29fb431594c84132e6e0091c 4b14e0f211ed6e2853a65270b04f3c72d39e627b
